src/Microsoft.Fx.Portability/ObjectModel/NuGetPackageInfo.cs

```diff
@@ -14,15 +14,25 @@ public class NuGetPackageInfo
         private bool _hashComputed;
         private int _hashCode;
 
+        public string PackageId { get; private set; }
+
+        // Dictionary of Framework/version of this package that is supported on that framework
+        public ImmutableDictionary<FrameworkName, string> SupportedVersions { get; private set; }
+
+        // AssemblyInfo is optional; it is present only when the package Id is "guessed" from the assembly name
         public string AssemblyInfo { get; private set; }
-        public FrameworkName Target { get; private set; }
-        public ImmutableList<NuGetPackageId> SupportedPackages { get; private set; }
 
-        public NuGetPackageInfo(string assemblyInfo, FrameworkName target, IEnumerable<NuGetPackageId> supportedPackages)
+        private static readonly HashSet<string> ImplicitlyReferencedPackages = new HashSet<string>() { "Microsoft.NETCore.App", "NETStandard.Library" };
```

Comment:
I believe package names are case insensitive. The comparer in this hashset could be stringcomparer.ignorecase

Reply:
Added comparer.